specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,561 @@
+      "required": [
+        "query",
+        "prediction"
+      ],
+      "properties": {
+        "query": {
+          "type": "string",
+          "description": "The conversation utterance given by the caller."
+        },
+        "detectedLanguage": {
+          "type": "string",
+          "description": "The system detected language for the query."
+        },
+        "prediction": {
+          "description": "The prediction result of a conversation project.",
+          "$ref": "#/definitions/BasePrediction"
+        }
+      }
+    },
+    "BasePrediction": {
+      "type": "object",
+      "description": "This is the base class of prediction",
+      "required": [
+        "projectKind"
+      ],
+      "discriminator": "projectKind",
+      "properties": {
+        "projectKind": {
+          "type": "string",
+          "description": "The type of the project.",
+          "enum": [
+            "conversation",
+            "workflow"
+          ],
+          "x-ms-enum": {
+            "name": "projectKind",
+            "modelAsString": true
+          }
+        },
+        "topIntent": {
+          "type": "string",
+          "description": "The intent with the highest score."
+        }
+      }
+    },
+    "OrchestratorPrediction": {
+      "type": "object",
+      "description": "This represents the prediction result of an Orchestrator project.",
+      "x-ms-discriminator-value": "workflow",
+      "required": [
+        "intents"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/BasePrediction"
+        }
+      ],
+      "properties": {
+        "intents": {
+          "description": "A dictionary that contains all intents. A key is an intent name and a value is its confidence score and target type. The top intent's value also contains the actual response from the target project.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/TargetIntentResult"
+          }
+        }
+      }
+    },
+    "TargetIntentResult": {
+      "type": "object",
+      "description": "This is the base class of an intent prediction",
+      "discriminator": "targetKind",
+      "required": [
+        "confidenceScore",
+        "targetKind"
+      ],
+      "properties": {
+        "targetKind": {
+          "type": "string",
+          "description": "This discriminator property specifies the type of the target project that returns the response.",
+          "enum": [
+            "luis",
+            "conversation",
+            "question_answering",
+            "non_linked"
+          ],
+          "x-ms-enum": {
+            "name": "targetKind",
+            "modelAsString": true
+          }
+        },
+        "apiVersion": {
+          "type": "string",
+          "description": "The API version used to call a target service."
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
```

Comment:
nit, this "confidenceScore" is double, while others is float. Is it intended?

Reply:
no, could be a typo. I changed it to float